1237-registrator_worklist/1238-creator_opens_my_messages.feature

```diff
@@ -1,12 +1,15 @@
 Feature: Creator opens My Messages
 
     @1238
-    Scenario: Creator opens My Messages
+    Scenario Outline: Creator opens My Messages
         Given that the user is logged in as Creator
         When they click the menu item My Messages
         Then they see the My Messages page
-        And they see a list of <MessageType> with fields <MessageTypeFields>
+        And they see a list of <MessageType> with fields:
+            | Publication title |
+            | Date              |
+            | Request status    |
         And they see that each item in the list is expandable
         Examples:
-            | MessageType | MessageTypeFields                           |
-            | DoiRequests | Publication Title, Date, DOI Request Status |
\ No newline at end of file
+            | MessageType |
+            | DoiRequests |
```

Comment:
grunnen til at det er opplisting av et element er pga det skal komme flere typer eller?

Reply:
jepp, bare `DoiRequest` i første runde, men gjorde det slik for at det skal bli enklere å legge til `ForApproval`, `Support`, etc senere